interface.go

```diff
@@ -23,6 +26,9 @@ type Subscriber interface {
 	// ExtendAckDeadline will postpone resending the given in-flight message for
 	// the specified duration
 	ExtendAckDeadline(ctx context.Context, messageID string, newDuration time.Duration) error
+
+	//DeleteTopic free all resources associated with subscription
```

Comment:
Update DeleteTopic to DeleteSubscription

Reply:
Thank you, will change